### PR TITLE
Ignore outdated JointTrajectory command

### LINKED ISSUE #53: Ignore outdated joint trajectory command
https://github.com/openspur/ypspur_ros/blob/39db1e5df2659865271dc429ff2175dd615c9fb1/src/ypspur_ros.cpp#L187

### PR DESCRIPTION
**Pull request by at-wat**
fix #53

### CODE REVIEW

**Comment by at-wat** [comment]
tested locally.
need to add a test.

**Comment by at-wat** [comment]
@seiga-k please take a look

File: src/ypspur_ros.cpp [hunk 1]
```diff
@@ -191,21 +191,33 @@ class YpspurRosNode
 #if !(YPSPUR_JOINT_ANG_VEL_SUPPORT)
     ROS_ERROR("JointTrajectory command is not available on this YP-Spur version");
 #endif
+    const ros::Time now = ros::Time::now();
+
     std_msgs::Header header = msg->header;
     if (header.stamp == ros::Time(0))
```
Comment (seiga-k) [inline src/ypspur_ros.cpp]: Is it correct to overwrite a timestamp?
I think it is more safe to take a timestamp.

Comment (seiga-k) [reply src/ypspur_ros.cpp]: Sorry, referenced line number is wrong. I comment about line 197 in new file.

Comment (at-wat) [reply src/ypspur_ros.cpp]: In ROS timestamp convention (especially in TF), zero timestamp means the latest possible time.
This is overwritten only if given timestamp is 0.

Comment (seiga-k) [reply src/ypspur_ros.cpp]: OK, I understand the thought.

**Pull request merged**